pkg/ring/kv/memberlist/memberlist_client_test.go

```diff
@@ -144,6 +146,14 @@ func (d dataCodec) Decode(b []byte) (interface{}, error) {
 	return out, err
 }
 
+func (d dataCodec) DecodeMultiKey(map[string][]byte) (interface{}, error) {
+	return nil, errors.New("memberlist does not support DecodeMultiKey")
```

Comment:
Maybe this is more a question to the proposal but 
does this mean Multikey will not be implemented in memberlist?

Reply:
@friedrichg we could implement memberlist with multikey in the future, but the biggest benefit of the multikey is for multi key storage. I am not sure how much memberlist would benefit from it as it is a gossip protocol.